defaults/aws_config.yaml

```diff
@@ -7,14 +7,13 @@ user_credentials_path: '~/.ssh/scylla-qa-ec2'
 instance_type_loader: 'c6i.xlarge'
 instance_type_monitor: 't3.large'
 # manual on creating loader AMI's: see docs/new_loader_ami.md
-ami_id_loader: 'scylla-qa-loader-ami-v20-ubuntu22'
+ami_id_loader: 'ami-07dc305835cc4ccbf'
```

Comment:
lets raise a task/issue for creating a new loader ami with tablets aware drivers

Reply:
https://github.com/scylladb/scylla-cluster-tests/issues/7663